test/commands/append.test.ts

```diff
@@ -0,0 +1,91 @@
+import {expect, test} from '@oclif/test'
+
+import { setMockCallback } from '../../../src/lib/client'
+import { buildResponse, expectRequest } from '../../helpers/http'
+const testToken = 'test-token'
+
+for(const appendType of ['fact', 'serial', 'atomic']) {
```

Comment:
All these commands are actually identical now, so I just fully re-used the test and run it 3 times.